src/main/webapp/app/exercises/modeling/manage/modeling-exercise-update.component.html

```diff
@@ -181,7 +181,101 @@ <h4 *ngIf="isImport" id="jhi-modeling-exercise-heading-import" jhiTranslate="art
+                        />
+                    </div>
+                </div>
+                <div class="col">
+                    <div class="form-group">
+                        <div>
+                            <label class="form-control-label" jhiTranslate="artemisApp.plagiarism.minimumScore">Minimum Score</label>
+                            <fa-icon [icon]="faQuestionCircle" placement="bottom auto" [ngbTooltip]="'artemisApp.plagiarism.minimumScoreTooltip' | artemisTranslate"></fa-icon>
+                        </div>
+                        <input
+                            type="number"
+                            required
+                            class="form-control"
+                            min="0"
+                            max="100"
+                            id="plagiarism-minimum-score"
+                            name="textExercise.plagiarismChecksConfig.minimumScore"
+                            #textExercise.plagiarismChecksConfig.minimumScore="ngModel"
+                            [(ngModel)]="modelingExercise.plagiarismChecksConfig!.minimumScore"
+                        />
+                    </div>
+                </div>
+                <div class="col">
+                    <div class="form-group">
+                        <div>
+                            <label class="form-control-label" jhiTranslate="artemisApp.plagiarism.minimumSize">Minimum Size</label>
+                            <fa-icon [icon]="faQuestionCircle" placement="bottom auto" [ngbTooltip]="'artemisApp.plagiarism.minimumSizeTooltip' | artemisTranslate"></fa-icon>
+                        </div>
+                        <input
+                            type="number"
+                            required
+                            class="form-control"
+                            min="0"
+                            max="100"
+                            id="plagiarism-minimum-size"
+                            name="textExercise.plagiarismChecksConfig.minimumSize"
+                            #textExercise.plagiarismChecksConfig.minimumSize="ngModel"
+                            [(ngModel)]="modelingExercise.plagiarismChecksConfig!.minimumSize"
+                        />
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div class="form-group col-auto border border-warning p-3">
+            <div class="row">
+                <div>
+                    <label jhiTranslate="artemisApp.exercise.continuousPlagiarismControl">Continuous Plagiarism Control</label>
+                    <fa-icon
+                        [icon]="faQuestionCircle"
+                        placement="bottom auto"
+                        [ngbTooltip]="'artemisApp.plagiarism.continuousPlagiarismControlTooltip' | artemisTranslate"
+                    ></fa-icon>
+                    <span class="badge rounded-pill text-bg-warning ms-1">BETA</span>
```

Comment:
It probably makes sense to extract this to a component so we don't have so search up how to implement it every time. Also, the stylesheet can then be confined to this single component.

Opinions? @Strohgelaender

Reply:
There is already a follow up issue for extracting plagiarism detection config to a separate component: #6869

Reply:
Alright,  then consider adding it to that. Would possibly simplify things in the future.